dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -383,11 +391,33 @@ def wait_for_element(selection_criteria, enabled)
   end
 end
 
+When /^I click on the link reading "([^"]*)"(?: within element "([^"]*)")?(?: to load a new (page|tab))?$/ do |text, parent, load|
+  link = nil
+  wait_short_until do
+    parent = @browser.find_element(:css, parent) if parent
+    parent ||= @browser
+    xpath = "//a[text()='#{text}']"
+    link = parent.find_element(:xpath, xpath)
+  end
+  page_load(load) {link.click}
+end
+
+Then /^the link reading "([^"]*)"(?: within element "([^"]*)")? goes to "([^"]*)"$/ do |text, parent, url|
+  link = nil
+  wait_short_until do
+    parent = @browser.find_element(:css, parent) if parent
+    parent ||= @browser
+    xpath = "//a[text()='#{text}']"
+    link = parent.find_element(:xpath, xpath)
+  end
+  expect(link.attribute("href")).to eq(replace_hostname(url)).or eq(url)
```

Comment:
I needed the `or` here because our test helpers will replace `forum.code.org` to `localhost-forum.code.org:3000` but our header uses a hard-coded URL... and that's a bit confusing to deal with